EssentialFeed/EssentialFeed/Comments/Comment API/RemoteCommentLoader.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  RemoteCommentLoader.swift
+//  EssentialFeed
+//
+//  Created by Robert Dates on 1/21/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+
+public class RemoteCommentLoader: CommentLoader {
+	
+	private let url: URL
+	private let client: HTTPClient
+	
+	public enum Error: Swift.Error {
+		case connectivity
+		case invalidData
+	}
+	
+	public init(client: HTTPClient, url: URL) {
+		self.url = url
+		self.client = client
+	}
+	public typealias Result = CommentLoader.Result
```

Comment:
Add missing line break

```suggestion
	}
	
	public typealias Result = CommentLoader.Result
```

Reply:
done